app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,72 @@
  *= require_tree .
  *= require_self
  */
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.flex {
+  display: flex;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
+}
+
+.main-container {
+  width: 60%;
+  margin-top: 2%;
+  gap: 5%;
+  flex-direction: column;
+  height: 100vh;
+}
```

Comment:
- [ ]  Good job so far, but you need to work a little bit on your design. You can see the parts that you need to work on in the screenshot below 👇 

Your design

<img width="439" alt="Screen Shot 2023-02-02 at 14 50 20" src="https://user-images.githubusercontent.com/74501523/216318664-4ce3395a-173a-4123-b7c4-c0e7fc62482d.png">

Wireframe

<img width="439" alt="Screen Shot 2023-02-02 at 14 50 32" src="https://user-images.githubusercontent.com/74501523/216318748-9094c412-f5b1-4e22-8ddd-ee7b3121772a.png">

- [ ] Also, don't forget to work a little bit on your design for the see-all post page 😉 😉 

Your design

<img width="439" alt="Screen Shot 2023-02-02 at 15 04 27" src="https://user-images.githubusercontent.com/74501523/216320332-8f6d29a8-1795-4b0d-ad7c-5629f0402ff2.png">

Wireframe

<img width="439" alt="Screen Shot 2023-02-02 at 15 04 44" src="https://user-images.githubusercontent.com/74501523/216320402-d36ce60d-e66a-43ad-9c90-f7f823818475.png">